app/src/main/res/values/strings.xml

```diff
@@ -270,7 +270,7 @@
     <string name="sort_by_times_opened">Times opened</string>
     <string name="sort_by_mobile_data">Mobile data</string>
     <string name="pref_import_export_blocking_rules">Import/export blocking rules</string>
-    <string name="pref_import_export_blocking_rules_msg">Import/export rules, import external rules from Watt or Blocker.</string>
+    <string name="pref_import_export_blocking_rules_msg">(Rules can be imported from Watt or Blocker.)</string>
```

Comment:
Because as an aside, it functions like a hint in its entirety.

Reply:
No, it denotes summary, not hint (you can take a look at the docs on android preferences). Consequently, there's no reason to add brackets.

Reply:
It is just a sidenote, not an explanation of what it does.

Reply:
It's not a side note, it's a summary of what this particular preference item can do.

Reply:
The summary is "Import/export blocking rules", an explanation would be something like "Blocking rules are entries meant to prevent loading of trackers or apps. (Can be imported from Watt or Blocker.)" Hence the brackets.

Reply:
“Import/export blocking rules” is the title and this string is the summary (not an explanation, hint or a side note). So, I suggest either to add a summary of what it does or revert back to the old one.